### PR TITLE
Adaptability added to 2nd step

### PR DESCRIPTION
**Pull request by FryDay969**
Before

900

690

601

600 less
600 less

After 

900

690

601

390

375

### CODE REVIEW

File: src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx [hunk 1]
```diff
@@ -60,14 +62,17 @@ const SubjectsStep = ({ stepLabel, btnsBox }) => {
   
   return (
     <Box sx={ styles.container }>
-      <Box sx={ styles.imgContainer }>
+      { isDesktop && (<Box sx={ styles.imgContainer  }>
```
Comment (MickhaelSh) [inline src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx:65]: move this container with image to variable, dry

**Review by MickhaelSh — COMMENTED** [review]
add screenshots before and after to your PR description, so we can see work that was done

File: src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx (hunk 1 above)
Comment (FryDay969) [reply src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx]: fixed

**Comment by FryDay969** [comment]
Screenshots have been added

**Review by MickhaelSh — APPROVED** [review]

**Review by Tolik170 — APPROVED** [review]

**Review by Yevhendzyuba — APPROVED** [review]

**Review by Yurenko — APPROVED** [review]

**Review by ArturBekhDEV — APPROVED** [review]

**Review by maarekua — APPROVED** [review]

**Pull request merged**